Remotion/Web/UnitTests/Core/ExecutionEngine/UrlMapping/UrlMappingLoaderTest.cs

```diff
@@ -0,0 +1,133 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System.Collections.Generic;
+using NUnit.Framework;
+using Remotion.Web.ExecutionEngine.UrlMapping;
+using Remotion.Web.UnitTests.Core.ExecutionEngine.TestFunctions;
+
+namespace Remotion.Web.UnitTests.Core.ExecutionEngine.UrlMapping
+{
+  [TestFixture]
+  public class UrlMappingLoaderTest
+  {
+    private class StubUrlMappingFileFinder : IUrlMappingFileFinder
```

Comment:
note: in the tests we usally have the nested classes at the bottom of the file because htey're not important.